plugins/pulp_ostree/plugins/importers/steps.py

```diff
@@ -110,11 +110,15 @@ def process_main(self, item=None):
         Add/update the remote summary information in the
         repository scratchpad.
         """
+        try:
+            lib_repository = lib.Repository(self.parent.storage_dir)
+            remote = lib.Remote(self.parent.repo_id, lib_repository)
+            refs = [r.dict() for r in remote.list_refs()]
+        except lib.LibError, le:
```

Comment:
It's better to use ``as`` instead of the comma for future compatibility with python 3.